### ISSUE TITLE
AP posts not bridging

### ISSUE THREAD

**Issue by hybridhavoc** [snarfed/bridgy-fed#1048]
I've noticed a few of my posts today on my Mastodon account not bridging over.

User page: https://fed.brid.gy/ap/@hybridhavoc@darkfriend.social

Sample post that worked previously: https://fed.brid.gy/log?path=%2Fqueue%2Fwebmention%2C%2Finbox&start_time=1715472839&key=https%3A%2F%2Fdarkfriend.social%2Fusers%2Fhybridhavoc%2Fstatuses%2F112425227434529934%2Factivity

Sample post that did not work today: https://fed.brid.gy/log?path=%2Fqueue%2Fwebmention%2C%2Finbox&start_time=1715559233&key=https%3A%2F%2Fdarkfriend.social%2Fusers%2Fhybridhavoc%2Fstatuses%2F112430889682458588%2Factivity

In comparing the logs things seem different here:

...
**I 2024-05-13 00:13:53.942322+00:00 Can't tell who this is from! Skipping followers.**
I 2024-05-13 00:13:53.995221+00:00 Wrote Key('Object', 'https://darkfriend.social/users/hybridhavoc/statuses/112430889682458588/activity') {'as2': '...', 'labels': ['activity'], 'object_ids': ['https://darkfriend.social/users/hybridhavoc/statuses/112430889682458588'], 'our_as1': '...', 'source_protocol': 'activitypub', 'status': 'ignored', 'type': 'post', 'users': [Key('ActivityPub', 'https://darkfriend.social/users/hybridhavoc')], 'new': True}
I 2024-05-13 00:13:53.995815+00:00 Returning 204: No targets, nothing to do ¯\_(ツ)_/¯
E 2024-05-13 00:13:53.996224+00:00 <class 'oauth_dropins.webutil.flask_util.NoContent'>: 204 No Content: No Content

**Comment by hybridhavoc** [comment]
I just attempted to unfollow and refollow @bsky.brid.gy@bsky.brid.gy to see if that would have any impact. While those actions are listed on my user page as listed above, the @bsky.brid.gy@bsky.brid.gy user profile now shows "Cancel follow" instead of "Unfollow", as though it's awaiting approval of a follow request.

**Comment by snarfed** [comment]
Thanks for the report, and for the sleuthing! Looks like this was indeed a regression. Easy fix, it'll be deployed in ~5m.

**Comment by hybridhavoc** [comment]
Unfortunately this does not appear to be solved on my end. After my previous posts and the fix the [user page](https://fed.brid.gy/ap/@hybridhavoc@darkfriend.social) has stopped registering any of my actions / posts on the Fediverse. The follow request was still in an unapproved state, with the "Cancel follow" option. I again thought perhaps I just needed to re-initiate, so I unfollowed and refollowed. This time those actions, too, were not registered on the user page and the new follow request is still in the same state, as though awaiting approval.

I also made just [a very basic test post](https://darkfriend.social/@hybridhavoc/112433967372466606) to see if that registered at all on the user page and it did not.
